app/views/govuk_publishing_components/components/_chat_entry.html.erb

```diff
@@ -7,12 +7,23 @@
   description_text ||= t("components.chat_entry.description")
   border_top ||= false
   border_bottom ||= false
+  disable_ga4 ||= false
 
   component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
   component_helper.add_class("gem-c-chat-entry")
   component_helper.add_class("gem-c-chat-entry--border-top") if border_top
   component_helper.add_class("gem-c-chat-entry--border-bottom") if border_bottom
   component_helper.add_class(shared_helper.get_margin_bottom)
+  component_helper.add_data_attribute({ module: "ga4-link-tracker" }) unless disable_ga4
+
+  unless disable_ga4
+    ga4_link_data = {}
+    ga4_link_data[:ga4_link] = {
+      event_name: "navigation",
+      type: "callout",
+      section: "GOV.UK Chat entry",
+  }.to_json
+  end
```

Comment:
Can you put the `ga4_link` straight into the object? E.g.

```ruby
ga4_link_data = {
  ga4_link: {
    # attributes in here
  }
}.to_json
```

Reply:
Cool, have updated